Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -29,6 +29,8 @@
 #include <Atom/RPI.Public/View.h>
 #include <Atom/RHI/DrawList.h>
 
+#include <Atom/RPI.Public/AssetQuality.h>
+
```

Comment:
consider placing this include next to the other Atom/RPI.Public include for better readability